### PR TITLE
[Product] Add Wishlist Product Entity

### PR DESCRIPTION
**Pull request by sofrie**
## Summary
- Add wishlist entity
- Add get product skeleton

## Test Plan
test manual in local and staging

### CODE REVIEW

**Comment by feechanz** [comment]
@sofrie 
please resolve conflict first

File: spring-command-pattern-entity/src/main/java/com/idspring/commandpattern/entity/Wishlist.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.idspring.commandpattern.entity;
+
+import lombok.*;
```
Comment (chandraarigunawan) [inline spring-command-pattern-entity/src/main/java/com/idspring/commandpattern/entity/Wishlist.java]: please don't use wildcard import

File: spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/ProductController.java [hunk 2]
```diff
@@ -15,5 +15,10 @@
   @Autowired
   private ServiceExecutor serviceExecutor;
 
-  
+  @RequestMapping(value = "/{productId}", method = RequestMethod.GET,
```
Comment (chandraarigunawan) [inline spring-command-pattern-web/src/main/java/com/idspring/commandpattern/controller/ProductController.java]: this method already added, pls create another method (put / delete / get all)

**Comment by feechanz** [comment]
I suggest u create new branch when want to create the PR

File: spring-command-pattern-entity/src/main/java/com/idspring/commandpattern/entity/Wishlist.java (hunk 1 above)
Comment (sofrie) [reply spring-command-pattern-entity/src/main/java/com/idspring/commandpattern/entity/Wishlist.java]: sure

**Review by chandraarigunawan — APPROVED** [review]
lgtm

**Pull request merged**